### DISCUSSION TITLE
How to jump to selection?

### DISCUSSION THREAD

**Discussion by diocletiann** [ibhagwan/fzf-lua#1423] [Q&A]
When I have a list of results, for example from "goto reference", I'd like the main window to automatically show the line of the currently selected result, and jump back to where I was before if I close the fzf-lua window without committing a selection. Basically the way Vertico works in Emacs. Thanks.

**Comment by ibhagwan** [comment]
No such option without customization, if you’re decent in lua and willing to put some effort you can make it happen if you use fzf-lua in a split using `winopts.split` and a custom previewer.

**Reply by diocletiann** [reply]
> No such option without customization, if you’re decent in lua and willing to put some effort you can make it happen if you use fzf-lua in a split using `winopts.split` and a custom previewer.

I'd be willing to try using a split. Do you provide a setting? Thanks.

**Reply by ibhagwan** [reply]
Look at `winopts.split`.
